composer.json

```diff
@@ -17,7 +17,7 @@
         "drupal/config_split": "^1.4",
         "drupal/config_update": "^1.6",
         "drupal/content_moderation_scheduled_updates": "^1.0",
-        "drupal/core": "8.6.12",
+        "drupal/core": "8.6.16",
```

Comment:
@GROwen could you double check that you merged hotfix updates back to develop as this should not be required.

Reply:
I'm not sure why this is showing as a diff @anthony-malkoun. The update to 8.6.16 is in both master and develop branches.

Reply:
@anthony-malkoun running a rebase has sorted it out.